cmd/buildah/addcopy.go

```diff
@@ -44,6 +44,9 @@ func addAndCopyCmd(c *cli.Context, extractLocalArchives bool) error {
 	}
 	name := args[0]
 	args = args.Tail()
+	if len(args) == 0 {
```

Comment:
This should be "== 1".  The command should be `buildah add containerID SRC [[...] DEST]`  So both args[0] and args[1] should be provided.  Good catch though, that definitely was a hole that needed filling!

Reply:
Ooops!  Sorry about that @q384566678, I somehow missed the Tail() call a line above.  Ouch.  This is fine.